test/functional/fixtures/api/es-next/native-dialogs-handling/test.js

```diff
@@ -1,14 +1,22 @@
-var errorInEachBrowserContains                = require('../../assertion-helper.js').errorInEachBrowserContains;
+var errorInEachBrowserContains                = require('../../../../assertion-helper.js').errorInEachBrowserContains;
 var getNativeDialogNotHandledErrorText        = require('./errors.js').getNativeDialogNotHandledErrorText;
 var getUncaughtErrorInNativeDialogHandlerText = require('./errors.js').getUncaughtErrorInNativeDialogHandlerText;
 
 
-var pageUrl        = 'http://localhost:3000/fixtures/native-dialogs-handling/pages/index.html';
-var pageLoadingUrl = 'http://localhost:3000/fixtures/native-dialogs-handling/pages/page-load.html';
-var pagePromptUrl  = 'http://localhost:3000/fixtures/native-dialogs-handling/pages/prompt.html';
+var pageUrl        = 'http://localhost:3000/fixtures/api/es-next/native-dialogs-handling/pages/index.html';
+var pageLoadingUrl = 'http://localhost:3000/fixtures/api/es-next/native-dialogs-handling/pages/page-load.html';
+var pagePromptUrl  = 'http://localhost:3000/fixtures/api/es-next/native-dialogs-handling/pages/prompt.html';
 
 
 describe('Native dialogs handling', function () {
+    it('Should remove dialog handler if `null` specified', function () {
```

Comment:
I think we should info about this in the documentation topic about `setNativeDialogHandler`.
It's possible now to pass `null` as an argument (see https://github.com/DevExpress/testcafe/pull/1315/files#diff-bc2ca31d9db34de5fa61db548150e9aaR192) /cc @VasilyStrelyaev